docs/src/examples/gallery.md

```diff
@@ -1,4 +1,6 @@
-# [PGFPlots manual gallery](@id manual_gallery)
+# [PGFPlots gallery](@id manual_gallery)
```

Comment:
```suggestion
# [PGFPlotsX gallery](@id manual_gallery)
```
maybe? Or just `Gallery`? Or maybe just something like `Miscellaneous examples`?

Reply:
Eh, with the sentence below its fine.